README.md

```diff
@@ -99,7 +99,9 @@ If you need to build Julia on a machine without internet access, use `make -C de
 
 **Note:** The build process will fail badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
 
-Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory). To run julia from anywhere you can:
+Once it is built, you can run the `julia` executable after you enter your julia directory and run
+    ./julia
```

Comment:
Note that this is Unix only.

Reply:
@yuyichao Are you suggesting to specify that this is valid only for unix? I am sorry but could not clearly get your point. Could you please elaborate it a bit? 
Thank you

Reply:
I'm just pointing out that this does not work on windows. You can either just mention that or give a way that works on windows too (e.g. `usr/bin/julia`/`usr/bin/julia-debug`) I don't know what people on windows prefer so I don't know what's better.